api-consumption/src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -663,15 +764,25 @@ private void reassignCommitted() {
         }
     }
 
-    void offsetChanged(final EventTypePartition key) {
-        if (offsets.containsKey(key)) {
-            final PartitionData data = offsets.get(key);
+    void offsetChanged(final EventTypePartition etp) {
```

Comment:
(unrelated) wonder why it's not `private`?

Reply:
used in test in the same package